br/pkg/restore/db.go

```diff
@@ -90,6 +90,11 @@ func (db *DB) ExecDDL(ctx context.Context, ddlJob *model.Job) error {
 		return errors.Trace(err)
 	}
 
+	// If query of the ddl job is empty or type of ddl job is in blacklist, ignore the ddl job.
+	if ddlJob.Query == "" || checkIsInActionList(ddlJob.Type, incrementalRestoreActionBlacklist) {
```

Comment:
`ExecDDL()` should be a universal interface,  don't check `checkIsInActionList()` in this function, I think.
Maybe put `checkIsInActionList()` in `FilterDDLJobs()`, or create a new function after  `FilterDDLJobs()`;
https://github.com/pingcap/tidb/blob/5a14cc3fd69c96a4a9c2348e4e87df7e49ae2ec1/br/pkg/restore/db.go#L361